NEW_RELEASE_NOTES.md

```diff
@@ -10,3 +10,5 @@ appropriate header in [RELEASE_NOTES.md](./RELEASE_NOTES.md).
 
 - engine: Support up to 4 side-by-side stereoscopic eyes, configurable at Engine creation time. See
   `Engine::Config::stereoscopicEyeCount`. [⚠️ **Recompile Materials**]
+- backend: Support importing native buffers as buffer objects.
+   Currently none of the backends support this.
```

Comment:
This release note sounds lame now, but obviously it will make more sense as Metal and OpenGL support are added